src/mgos_onewire.c

```diff
@@ -74,26 +75,37 @@ static bool onewire_wait(struct mgos_onewire *ow, uint32_t n) {
     if (n == 0) return false;
     mgos_usleep(2);
     n--;
-  } while (!mgos_gpio_read(ow->pin));
+  } while (!mgos_gpio_read(ow->pin_in));
   return n != 0;
 }
 
+static void onewire_prepare_input(struct mgos_onewire *ow) {
```

Comment:
because the whole thing is timing-sensitive, we want this inlined.
compiler is probably smart enough to inline it anyway, but let's be explicit about it.
please add `inline` to the function declaration.